example/javalib/linting/3-palantirjavaformat/build.mill

```diff
@@ -0,0 +1,23 @@
+package build
+import mill._, javalib._
+import $ivy.`com.lihaoyi::mill-contrib-palantirjavaformat:`
+
+import mill.contrib.palantirjavaformat._
+
+object `package` extends RootModule with PalantirJavaFormatModule {
+}
+
+/** Usage
+
+> ./mill javafmt --check                # check Java source files for formatting errors
+...: checking format in java sources ...
+.../src/A.java
+...: palantirjavaformat aborted due to format error(s) (or invalid plugin settings/palantirjavaformat options)
+
+> ./mill javafmt src/A.java             # format Java source file
+
+> ./mill javafmt                        # alternatively, format all Java source files
+
+*/
```

Comment:
We should also show off the `./mill mill.contrib.palantirjavaformat.PalantirJavaFormat/` workflow, and after formatting is run we should run `check` again and ensure it does not error

Reply:
Updated. Please review.

Reply:
Had to remove `check` examples.
```sh
 Incomplete argument --check <bool> is missing a corresponding value
  Expected Signature: palantirformat
    --check <bool>
    sources <str>...
  

  ,)
```